components/CreateEventForm/CreateEventForm.js

```diff
@@ -50,7 +50,7 @@ export function CreateEventForm(props) {
   }
 
   const verifyTotalPlayersIsGreaterThanAttending = () => {
-    if (playersRequired <= currentlyAttending) {
+    if (parseInt(playersRequired) <= parseInt(currentlyAttending)) {
```

Comment:
Cool! This is a quick fix for the problem! I wonder why we were getting this error only when using double digit string numbers? It may have that some type coercion was happening where if the string was just one character that was a number, it could convert it to a number data type, but if it was two characters, it couldn't make it a number??